drivers/npmx/npmx_shell.c

```diff
@@ -4170,7 +4122,7 @@ SHELL_STATIC_SUBCMD_SET_CREATE(
 	SHELL_CMD(charger, &sub_charger, "Charger", NULL),
 	SHELL_CMD(errlog, &sub_errlog, "Reset errors logs", NULL),
 	SHELL_CMD(gpio, &sub_gpio, "GPIO", NULL), SHELL_CMD(ldsw, &sub_ldsw, "LDSW", NULL),
-	SHELL_CMD(leds, &sub_leds, "LEDs", NULL), SHELL_CMD(pof, &sub_pof, "POF", NULL),
+	SHELL_CMD(led, &sub_led, "LED", NULL), SHELL_CMD(pof, &sub_pof, "POF", NULL),
```

Comment:
`led` should be before `ldsw`

Reply:
I told you wrong, my mistake. For some reason, I thought that 'e' is before 'd' in the alphabet. Sorry 😁